### PR TITLE
Added card Sonu Raghuwanshi(Update index.html)

### PR DESCRIPTION
**Pull request by sonuraghuwanshi**
This is my first contribution. Hope I have met the expectations. Looking forward to contribute.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11347,6 +11347,41 @@ <h2>A project for first-time contributions</h2>
...
+                        <p>3 Useful Dev Resources</p>
+                        <ul>
+                            <li>
+                                <a href="#" target="_blank" title="https://www.hackerearth.com/">
+                                    Hacker-Earth
+                                </a>
+                            </li>
+                            <li>
+                                <a
+                                    href="#" target="_blank" title="https://stackoverflow.com/">
```
Comment (uaparicio) [inline index.html:11371]: Same than above

File: index.html [hunk 2]
```diff
@@ -11347,6 +11347,41 @@ <h2>A project for first-time contributions</h2>
...
+                                    Stack Overflow
+                                </a>
+                            </li>
+                            <li>
+                                <a href="#" target="_blank" title="https://www.learnpython.org/">
```
Comment (uaparicio) [inline index.html:11376]: Same than above

File: index.html [hunk 3]
```diff
@@ -11347,6 +11347,41 @@ <h2>A project for first-time contributions</h2>
...
+                        <a href="https://twitter.com/sonu8823812809" target="_blank"> @Sonu Raghuwanshi </a>
+                    </p>
+                    <p class="about">
+                        I am a Competitive Programming. I love problem solving. I am fond of travelling.
+                    </p>
+                    <div class="resources">
+                        <p>3 Useful Dev Resources</p>
+                        <ul>
+                            <li>
+                                <a href="#" target="_blank" title="https://www.hackerearth.com/">
```
Comment (uaparicio) [inline index.html:11365]: Please, make sure you include the URL into the href attribute, i.e.:
`<a href="https://www.hackerearth.com/" target="_blank" title="Hacker Earth">
      Hacker Earth
  </a>
`

**Comment by sonuraghuwanshi** [comment]
I have changed the files. Please check and merge

**Pull request merged**